app/model/Forms.scala

```diff
@@ -146,21 +149,39 @@ object Forms {
       su.setInclusiveHostCount(true)
       su.getInfo
     }
-    val whitelistedIpConstraint: Constraint[String] = Constraint({
-      whitelistedIp =>
-        Try(new SubnetUtils(whitelistedIp)) match {
-          case Failure(_) => Invalid(Seq(ValidationError("whitelistedIp.invalid")))
-          case _ =>
-            val ipAndMask = whitelistedIp.split("/")
-            if (privateNetworkRanges.exists(_.isInRange(ipAndMask(0)))) Invalid(Seq(ValidationError("whitelistedIp.invalid.private")))
-            else if (ipAndMask(1).toInt < 24) Invalid(Seq(ValidationError("whitelistedIp.invalid.range")))
-            else Valid
-        }
+    val whitelistedIpsConstraint: Constraint[String] = Constraint({
+      whitelistedIps => toSetOfWhitelistedIps(whitelistedIps).map(validateWhitelistedIp).fold(Valid)(reduceValidationResults)
     })
 
-    val form = Form(mapping(
-      whitelistedIp -> text.verifying(whitelistedIpConstraint).verifying("whitelistedIp.required", _.nonEmpty)
-    )(WhitelistedIpForm.apply)(WhitelistedIpForm.unapply))
+    def reduceValidationResults(a: ValidationResult, b: ValidationResult): ValidationResult = {
+      (a, b) match {
+        case (Valid, Valid) => Valid
+        case (Valid, i: Invalid) => i
+        case (i: Invalid, Valid) => i
+        case (Invalid(e1), Invalid(e2)) => Invalid(e1 ++ e2)
+      }
+
+    }
+
+    def validateWhitelistedIp(whitelistedIp: String): ValidationResult = {
+
+      Try(new SubnetUtils(whitelistedIp)) match {
+        case Failure(_) => Invalid(Seq(ValidationError("whitelistedIp.invalid", whitelistedIp)))
+        case _ =>
```

Comment:
Do we need to validate that the user hasn't included 127.0.0.1(/32) or 255.255.255.255(/32)?

Reply:
That could be a nice to have.